libs/dh/market-participant/actors/feature-organizations/src/lib/dh-organizations-overview.component.html

```diff
@@ -40,4 +40,6 @@ <h3>{{ t("organizations") }}</h3>
   </vater-flex>
 </watt-card>
 
-<dh-organization-drawer [id]="id()" (closed)="navigate(null)" />
+<dh-organization-details [id]="id()" (closed)="navigate(undefined)" />
```

Comment:
issue: since the component is related to the `details/:id` a route, it shoudn't be included in here too. Angular will take care of instantiating it. That also means that the `id` should be a property of the component (which already is), where it's value is taken from the route params. The `closed` event should be handled in the component instead of emitting it.

Reply:
Good find, it also fixed the issue with the edit modal not getting closed ! :-)